src/current/v24.3/set-up-logical-data-replication.md

```diff
@@ -0,0 +1,189 @@
+- `validated`: Inserts data at the SQL layer. All SQL constraints are adhered to. If a row is unable to apply due to a SQL constraint, it will be put in the [dead letter queue (DLQ)]({% link {{ page.version.version }}/manage-logical-data-replication.md %}#dead-letter-queue-dlq) for manual resolution. SQL does not recognize deletion records used by [MVCC]({% link {{ page.version.version }}/architecture/storage-layer.md %}#mvcc), therefore an update to the same key from cluster A will successfully apply on cluster B even if that key was deleted on cluster B before the cluster A update was replicated.
+
+{{site.data.alerts.callout_info}}
+If you would like to ignore TTL deletes in the LDR stream, you can use the `discard = ttl-deletes` option in the `CREATE LOGICAL REPLICATION STREAM` statement.
+{{site.data.alerts.end}}
+
+1. From the destination cluster, start the LDR stream. Use the fully qualified table name and ensure the source and destination table names are identical:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE LOGICAL REPLICATION STREAM FROM TABLE {database.public.table_name} ON 'external://{source_external_connection}' INTO TABLE {database.public.table_name};
+    ~~~
+
+    You can change the default `mode` using the `WITH mode = validated` syntax.
+
+    Once the LDR stream has started, an LDR job will start on the destination cluster. You can [pause]({% link {{ page.version.version }}/pause-job.md %}), [resume]({% link {{ page.version.version }}/resume-job.md %}), or [cancel]({% link {{ page.version.version }}/cancel-job.md %}) the LDR job with the job ID. Use `SHOW LOGICAL REPLICATION JOBS` to display the LDR job IDs:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SHOW LOGICAL REPLICATION JOBS;
+    ~~~
+    ~~~
+            job_id        | status  |          targets          | replicated_time
+    ----------------------+---------+---------------------------+------------------
+    1012877040439033857   | running | {database.public.table}   | NULL
+    (1 row)
+    ~~~
+
+    If you're setting up bidirectional LDR streams, both clusters will have a history retention job and an LDR job running.
+
+1. Move on to [Step 4](#step-4-optional-set-up-bidirectional-ldr-streams) to set up a second LDR stream. Or, once you have set up your required LDR streams, refer to [Step 5](#step-5-monitor-the-ldr-jobs) to monitor the jobs in the DB Console.
+
+## Step 4. (Optional) Set up bidirectional LDR streams
+
+At this point, you've set up one LDR stream from cluster A as the source to cluster B as the destination. 
```

Comment:
More general comment, do we want to start talking about all the clusters as in Cluster A and Cluster B, or stick with source/dst?

Reply:
I think because we're doing bidirectional, I think it makes sense to keep both. It is hard to describe using just A and B, because that is pretty abstracted away from the direction flow. However, if you then try to just describe in terms of source/destination, you'll run into situations where you have to say "From the original destination cluster", which gets pretty ugly, very quickly.